Added gitignore and Pools for Azure ML (#20)
proge-software/uninapoli-csml: 02.azure-ml/polls/polls.en.md

﻿# Polls 

## Introduction 

> ### Which python libraries have you already used? Answer with 👍

Reviewer: ```suggestion
### Which ones of the following python libraries have you already used? Answer with 👍
```

- [ ] Pandas
- [ ] NumPy
- [ ] Matplotlib
- [ ] Seaborn
- [ ] Scikit-learn

> ### What is your degree of knowledge on Jupyter Notebook? Answer with 👍

Reviewer: **on** Jupyter Notebook or **of** Jupyter Notebook?

```suggestion
### What is your degree of knowledge of Jupyter Notebook? Answer with 👍
```

- [ ] Never heard about it
- [ ] I've heard about it, but I've never used it
- [ ] I've played with sometimes
- [ ] I'm a Jupyter hero

### Have you ever made forecast analysys? Answer with 👍

- [ ] Never did in my life
- [ ] I only have read about it
- [ ] I did, but with statistic

> - [ ] I did with machine learning

Reviewer: I'd recommend to remove this difference between statistic/ML

## Azure Machine Learning Studio (Preview)

### Have you ever dealt with Virtual Machines? Answer with 👍

- [ ] Never in my life
- [ ] I've virtualized Windows / Linux / MacOS in my PC
- [ ] I've used a VPS (Virtual Private Server) to deploy my code online
- [ ] I've dealts with a cluster of virtual machine / Kubernetes

## Where to go next

> ### Wich deep learning libraries / framework have you already heard about it? Answer with 👍

Reviewer: Which is misspelled and remove the "it" at the end

```suggestion
### Which deep learning libraries / framework have you already heard about? Answer with 👍
```
